cime/config/e3sm/machines/config_batch.xml

```diff
@@ -398,7 +398,8 @@
 
    <batch_system MACH="compy" type="slurm">
     <queues>
-      <queue walltimemax="00:59:00" default="true">slurm</queue>
+      <queue walltimemax="00:30:00" nodemax="40" default="true">short</queue>
```

Comment:
30 minutes is quite a small walltimemax. How was this number determined?

Reply:
This is the default time the queue should be using, the max is 2hrs. Is there a way to mention the default time? I just saw walltimemax and walltimemin options there.

Reply:
@singhbalwinder , CIME is good at determining a reasonable walltime selection for jobs. This field is for MAX walltime and should be treated as such.

Reply:
Thanks. I will change it to 2hrs.

Reply:
I have modified walltimemax values for both queues